Modules/Core/Transform/test/itkEuler3DTransformTest.cxx

```diff
@@ -34,10 +34,34 @@ itkEuler3DTransformTest(int, char *[])
   using EulerTransformType = itk::Euler3DTransform<double>;
   auto eulerTransform = EulerTransformType::New();
 
+  // Testing preservation of the center of rotation
```

Comment:
Did you consider using GoogleTest?  It's really convenient, as it allows you to specify what you "expect" in just a few lines of code:  

    eulerTransformWithCenter->SetCenter(centerOfRotation);
    eulerTransformWithCenter->SetIdentity();
    EXPECT_EQ(eulerTransformWithCenter->GetCenter(), centerOfRotation);

Anyway, maybe something to consider for the next time  😃